fake-files/output-expected/wang.cells.json

```diff
@@ -1,7 +1,25 @@
 {
  "12": {
   "mappings": {},
-  "genes": {},
+  "genes": {
+   "ADAM9": 2,
+   "AHCY": 2,
+   "AKAP11": 1,
+   "ATN1": 2,
+   "ATP1A1": 2,
+   "ATP6AP1": 2,
+   "BRCA2": 2,
+   "CDC45": 2,
+   "COLGALT1": 2,
+   "CRKL": 2,
+   "CSDE1": 2,
+   "CSE1L": 2,
+   "DAP": 2,
+   "DDB1": 2,
+   "DDX21": 2,
+   "DHX9": 2,
+   "DNAJB1": 2
+  },
```

Comment:
Have you been able to see how this renders? If we could have a `vitessce` branch that pointed to the data after running this script, that would help us say if everything is right.